pixel/style.css

```diff
@@ -85,3 +85,7 @@ Tags: one-column, custom-colors, custom-menu, custom-logo, editor-style, feature
 	font-weight: 300;
 	line-height: 1.6;
 }
+
```

Comment:
```suggestion
// Necessary so the header containing the navigation fills the entire remaining horizontal space, since it is positioned in a flex group with the site logo.
```